sway-lib-std/src/error_signals.sw

```diff
@@ -1,18 +1,18 @@
 //! Values which signify special types of errors when passed to `std::revert::revert`.
 library;
 
-/// Revert with this value for a failing call to `std::revert::require`.
-/// 18446744073709486080
+/// A revert with this value signals that it was caused by a failing call to `std::revert::require`.
+/// The value is: 18446744073709486080
```

Comment:
I'd probably put this under additional information

Reply:
The value?

Reply:
```suggestion
/// A revert with this value signals that it was caused by a failing call to `std::revert::require`.
/// 
/// # Additional Information
///
/// The value is: 18446744073709486080
```